tools/quota-monitoring-alerting/python/requirements.txt

```diff
@@ -0,0 +1,54 @@
+asgiref==3.3.4
```

Comment:
All Source files should have a License Header. Please add one

Reply:
Done